### PR TITLE
Update 03-generators.mdx

### PR DESCRIPTION
**Pull request by omar-dulaimi**
## Describe this PR

added:

- class validator generator
- zod generator
- trpc generator

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 1]
```diff
@@ -77,3 +77,4 @@ The following is a list of community created generators. If you want to create y
 - [`zod-prisma`](https://github.com/CarterGrimmeisen/zod-prisma): Creates Zod schemas from your Prisma models.
 - [`prisma-pothos-types`](https://github.com/hayes/pothos/tree/main/packages/plugin-prisma): Makes it easier to define Prisma-based object types, and helps solve n+1 queries for relations. It also has integrations for the Relay plugin to make defining nodes and connections easy and efficient.
 - [`graphql-schema-generator`](https://github.com/prisma-korea/graphql-schema-generator): Generates GraphQL schema from Prisma schema so that you can use any GraphQL tool you want.
+- [`prisma-class-validator-generator`](https://github.com/omar-dulaimi/prisma-class-validator-generator): Emit typescript models from your prisma schema with class validator validations ready.
```
Comment (Jolg42) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
- [`prisma-class-validator-generator`](https://github.com/omar-dulaimi/prisma-class-validator-generator): Emit TypeScript models from your Prisma schema with class validator validations ready.
```

**Review by Jolg42 — APPROVED** [review]

**Review by janpio — APPROVED** [review]

**Comment by omar-dulaimi** [comment]
Hello,

Should I submit other generators I built?

Or should I wait until this one gets merged?

**Comment by Jolg42** [comment]
@omar-dulaimi I think you can update this PR here 👍🏼

**Comment by omar-dulaimi** [comment]
@Jolg42 I have updated this PR with two more generators.

**Pull request merged**